ng-dev/pr/common/fetch-pull-request.ts

```diff
@@ -134,13 +134,16 @@ export const PR_SCHEMA = {
 
 export type PullRequestFromGithub = typeof PR_SCHEMA;
 
-export const PR_FILES_SCHEMA = {
-  nodes: [
-    {
-      path: graphqlTypes.string,
-    },
-  ],
-};
+export const PR_FILES_SCHEMA = params(
```

Comment:
is this needed with params already defined below?

Reply:
Yes, this is passing in the initial params for the query used for the pagination call.